store/src/java/com/zimbra/util/ExternalVolumeInfoHandler.java

```diff
@@ -271,9 +317,11 @@ public void editGlobalConfigOnDeleteVolume(int volumeId, String serverId) throws
                    globalExtStoreConfJson.put(GlobalExternalStoreConfigConstants.A_S3_UNIFIED_VOLUME, unifiedJsonObject);
                    Provisioning.getInstance().getConfig().setGlobalExternalStoreConfig(globalExtStoreConfJson.toString());
                 }
+                //flushCache(zsc);
```

Comment:
commented code

Reply:
The approach was under discussion in the draft PR. Uncommented it. Thanks!